packages/react/src/combobox/ComboboxBase/ComboboxBase.tsx

```diff
@@ -202,14 +215,26 @@ export function ComboboxBase<Option extends DefaultComboboxOption>({
 									<Fragment>
 										{inputItems?.length ? (
 											inputItems.map((item, index) => (
-												<ComboboxListItem
+												<li
```

Comment:
Rendering a raw `li` instead of the `ComboboxListItem` component for speed and because we already have `isIos` hook in this component